### ISSUE TITLE
Winning probability

### ISSUE THREAD

**Issue by zsoerenm** [philihp/openskill.js#64]
I finally found some time to invest in openskill and I had a look into your benchmarks. Great results!
I have a question with respect to the accuracy:
You calculate the correctness solely based on the mu value (https://github.com/philihp/openskill.js/blob/master/benchmark/load.js#L67).
Sublee (author of the Python-Trueskill implementation) suggests to calculate the winning probability based on both mu and sigma: https://trueskill.org/#win-probability
Is there any reason behind this? 
I am curious to hear your thoughts about this.

**Comment by philihp** [comment]
Awesome! Mainly for simplicity. I was aggregating the team to find the most likely outcome and then compare that to the actual.

Using the OverTrack data, we could/should actually compare how this would change things. TODO :)

**Comment by philihp** [comment]
I ran a benchmark with this on the Halo2 Head-to-Head data from TrueSkill (https://github.com/dotnet/mbmlbook/tree/master/src/3.%20Meeting%20Your%20Match/Data) and didn't find any difference with either methods, with either TrueSkill or Openskill. I only tried to predict games where at least 1 player had at least 1 prior match, and in all 4 cases it was incorrect 2305 (39.415%) out of 5848 times.

Getting a predicted ranking of teams in an n-team game is pretty simple if you just sort by `mu`, or by `mu - 3*sigma` (e.g. as in ordinal), just benchmark it with some test data.
